crates/topos-tce-storage/src/fullnode/mod.rs

```diff
@@ -135,6 +140,12 @@ impl WriteStore for FullNodeStore {
         // Adding certificate to target_streams
         // TODO: Add expected position instead of calculating on the go
```

Comment:
Is this TODO still relevant?

Reply:
Somehow yes as I want to investigate if we could also have expected position in the target streams